File: C&C++/VIjay_Joshi/Day1/ques2.cpp
#include <iostream>
using namespace std;


// 2. Write a program to calculate the sum of the digits of a
//  three-digit number entered by the user.

int main(){
    int number;
    cout<<"Enter the 3 digit number"<<endl;
    cin>>number;
    if(number>=1000 ||number<100 ){
        cout<<"Please only Enter 3 digit number"<<endl;
        return 0;
    }

    int sum=0;
   int temp=number;
    for(int i=0;i<3;i++){
    sum+=temp%10;
    temp=temp/10;
    }
cout<<"sum of digits is: "<<sum;
  return 0;
}

File: Python/Ansh Gulati/Day-3_05-07-23/Ques-2.py
''' Ques- Write a Python program that prompts the user to enter a 
series of integers, one at a time, until they enter a negative number.
The program should then terminate and compute and print the sum of all 
the positive integers entered by the user.'''

sum=0
number=0
while(number>=0):
    number=int(input("Enter a Positive Number: "))
    if(number>=0):
        sum=sum+number
    else:
        break

print(sum)
